db/data/development-data/index.js

```diff
@@ -1,4 +1,6 @@
-exports.articleData = require('./articles.js');
-exports.commentData = require('./comments.js');
-exports.topicData = require('./topics.js');
-exports.userData = require('./users.js');
+exports.articleData = require("./articles.js");
+exports.commentData = require("./comments.js");
+exports.topicData = require("./topics.js");
+exports.userData = require("./users.js");
+
+module.exports = { topicData, articleData, userData, commentData };
```

Comment:
you don't actually need this line here as they all have `exports.` attached to them :)